test/unit/self_collectives.c

```diff
@@ -82,37 +83,76 @@ int main(void) {
     /* Note: Broadcast does not modify the output buffer at the root */
     if (me == 0) printf(" + broadcast\n");
 
+    /* Set up active set team (start=me, stride=1, size=1) for all tests*/
+    shmem_team_split_strided(SHMEM_TEAM_WORLD, me, 1, 1, NULL, 0, &new_team);
```

Comment:
Minor thing - we don't need `new_team` and the call to split if we are not enabling deprecated tests... could it move inside and `#else` block?  Also, the declaration of `new_team` doesn't need to be global, right?

This one is probably safe to ignore... just wondering if it's cleaner.